trpcultivate_phenotypes/tests/src/Kernel/ServiceTermTest.php

```diff
@@ -56,32 +59,17 @@ protected function setUp(): void {
     // Set test environment.
     \Drupal::state()->set('is_a_test_environment', TRUE);
 
+    // Create a test chado instance and then set it in the container for use by our service.
```

Comment:
Introduces a standards error (too long ;-p). This code suggestions fixes it.

```suggestion
    // Create a test chado instance as needed by our service.
```

Reply:
The code suggestion does not cause Problem and is committed.